app/cdap/components/AbstractWidget/FormInputs/TextBox/index.tsx

```diff
@@ -72,6 +80,9 @@ const TextBoxComponent: React.FC<ITextBoxProps> = ({
       onKeyPress={onKeyPress}
       placeholder={placeholder}
       readOnly={disabled}
+      multiline={multiline}
+      minRows={customMinRows ? customMinRows : defaultProps.minRows}
```

Comment:
This and maxRows only apply when multiline is true (won't affect existing text boxes).